libs/common/src/archive.rs

```diff
@@ -35,3 +36,41 @@ pub fn archive<P: AsRef<Path>>(folder: P) -> Result<Option<String>> {
         Ok(Some(STANDARD.encode(bytes)))
     }
 }
+
+pub fn archive_entries(dir_walker: Walk, suffix: Option<&Path>) -> Result<Option<String>> {
+    let encoder = GzEncoder::new(Vec::new(), Compression::default());
+    let mut tar = tar::Builder::new(encoder);
+
+    // skip non-relevant fs metadata
+    tar.mode(tar::HeaderMode::Deterministic);
```

Comment:
does this ignore also file like [.DS_Store](https://en.wikipedia.org/wiki/.DS_Store)?

Reply:
No I don't think so. But they are usually added to gitignore.